<launch>
  <arg name="auto_sort" default="false"/>

  <include file="$(find widowx_block_manipulation)/launch/block_manipulation.launch"/>

  <node name="block_manipulation_demo" pkg="widowx_block_manipulation" type="block_manipulation_demo" output="screen" >
    <param name="arm_link" value="/arm_base_link" />
    <param name="gripper_open" value="0.031" />
    <param name="block_size" value="0.02" />
    <param name="gripper_tighten" value="0.004" />
    <param name="z_up" value="0.16" />
    <param name="target_bin_height" value="0.14" />
    <param name="table_height" value="0.0" />
  
    <param name="auto_sort" value="$(arg auto_sort)" />

  </node>
</launch>